src/View/Router.ts

```diff
@@ -39,20 +39,35 @@ export class Router {
   }
 
   onNewClaim = async (message: any) => {
+    const logger = this.logger.child({ method: 'onNewClaim' })
+
     const messageContent = message.content.toString()
+    logger.info({ messageContent }, 'Setting message content on works')
```

Comment:
Why did you add this message at `info` level?

Reply:
well, just I follow the same pattern that I saw here

https://github.com/poetapp/node/blob/b2d4180edb9b0018f9faf0a5cd3f126c181c7da4/src/View/Router.ts#L127-L129

I thought you were logging the inputs before of these been used in the call

what do you say?

Reply:
Got it.

I haven't formally defined what logging levels we should use in each scenario yet, so for now can you change them to `trace` to be on the safe side?

I don't want the `info` level to be too verbose.